PlayFieldBuddy.Api/Controllers/UserController.cs

```diff
@@ -26,7 +26,7 @@ public UserController(IUserService userService, IUserRepository userRepository,
     
     
     [HttpGet("{id}")]
-    public async Task<IActionResult> GetSingleUser(Guid id, CancellationToken cancellationToken)
+    public async Task<ActionResult> GetSingleUser(Guid id, CancellationToken cancellationToken)
```

Comment:
imo should be IActionResult because is async action. https://learn.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-8.0 <- But i'm open for disscussion, because I'm not sure xD